### PR TITLE
Added IOS and IOS XE

### PR DESCRIPTION
**Pull request by paulhoogsteder**

### CODE REVIEW

File: guides/small_prefixes.md [hunk 1]
```diff
@@ -50,3 +50,24 @@ prefix set too_small; {
     return true;
 }
 ```
+
+## Cisco classic IOS and IOS XE
+```
+ip prefix-list peerfilter seq 5 deny 0.0.0.0/0
+ip prefix-list peerfilter seq 10 permit 0.0.0.0/0 ge 8 le 28
```
Comment (job) [inline guides/small_prefixes.md]: i think `le 24` makes more sense

Comment (paulhoogsteder) [reply guides/small_prefixes.md]: All the other examples use le 27 or le28; and RIPE has given out several PI blocks size 25-27. So does the text above the examples.

Comment (paulhoogsteder) [reply guides/small_prefixes.md]: No my mistake, the examples don't - but still I think le 27 makes more sense. But I can change it anyhow.

**Comment by job** [comment]
> No my mistake, the examples don't - but still I think le 27 makes more sense. But I can change it anyhow.

I updated the examples to reflect reality. You are right that some RIRs proclaim the right to assign 'longer then /24' to customers, but I (and tons of global operators based on the filters commonly used) view IPv4 address shortage as insufficient of a reason to weaken sane filters.

**Pull request merged**